configs/ARM_MCS_verified.cmake

```diff
@@ -22,4 +22,5 @@ set(KernelPrinting OFF CACHE BOOL "")
 set(KernelNumDomains 16 CACHE STRING "")
 set(KernelMaxNumBootinfoUntypedCap 166 CACHE STRING "")
 set(KernelIsMCS ON CACHE BOOL "")
+set(KernelStaticMaxBudgetUs "(60 * 60 * MS_IN_S * US_IN_MS)" CACHE STRING "")
```

Comment:
It looks like this changes the permissions of this file to `0755`. Can you make sure the permissions of this file in this commit remain `0644`?

Reply:
Where do you see that? I fixed it up before I created the PR

Reply:
That is what is currently failing in they style checks.

Reply:
Might be an issue with `style.sh` which doesn't seem to like CMake files being executable although you made the verified configurations executable scripts a while back.